Library/Homebrew/keg.rb

```diff
@@ -208,6 +208,7 @@ def ==(other)
 
   def empty_installation?
     Pathname.glob("#{path}/**/*") do |file|
+      return false if file.executable?
       next if file.directory?
       basename = file.basename.to_s
       next if Metafiles.copy?(basename)
```

Comment:
Oh, I see! Metafiles such as `readme` are under root directory not subdirectories. I will fix `#empty_installation?` to check metafiles under only root directory and add tests for it.